client/dashboard/components/section-controls/style.scss

```diff
@@ -0,0 +1,14 @@
+/** @format */
+
+.woocommerce-section-controls {
+	border-top: $border-width solid $core-grey-light-500;
+
+	.dashicon {
+		margin: 0 $gap-smaller 0 0;
+		vertical-align: bottom;
+	}
+
+	.woocommerce-ellipsis-menu__item div {
```

Comment:
What would you think about rewriting this CSS ruleset, so we don't need the wrapping `<div>`?
```CSS
.woocommerce-ellipsis-menu__item {
	padding-bottom: 12px;
}
```